src/Build/Resources/xlf/Strings.es.xlf

```diff
@@ -191,7 +191,7 @@
       </trans-unit>
       <trans-unit id="OM_MatchOnMetadataIsRestrictedToReferencedItems">
         <source>Only item types may be referenced when removing with MatchOnMetadata.</source>
-        <target state="new">Only item types may be referenced when removing with MatchOnMetadata.</target>
+        <target state="translated">Solo se puede hacer referencia a los tipos de elemento al quitar con MatchOnMetadata.</target>
```

Comment:
...a los objetos de tipo elemento...?

Elemento sounds generic and like it might be confused with XML elements.

Reply:
Feedback incorporated (399008)
This will take few days to get back and will come as a new PR with othe cumulative translations.
You could:
a. Approve and complete this PR. The next PR (Friday) will have a subset of the new updates/
b. Close this and the new PR will have the new subset plus this.